Build Redesign of Temp Page

jasminepvo opened the pull request:
Built out the redesigned temp page with responsiveness. Please double check the responsiveness. The media queries are set at 
- Phone: @media screen and (max-width: 767px)
- Tablet: @media screen and (min-width: 768px) and (max-width: 1023px)
![Screen Shot 2022-10-28 at 8 48 09 PM](https://user-images.githubusercontent.com/99847030/198754221-150a54fe-7edb-47da-9c7f-68744c76b8c9.jpg)

Also made adjustments to the html title, description and favicon as the logo
![Screen Shot 2022-10-28 at 8 51 55 PM](https://user-images.githubusercontent.com/99847030/198754367-34a0bc0e-f8a1-417e-bf88-2e6701a94d25.jpg)

HotSoupRepos committed:
Merge pull request #2 from HotSoupRepos/dev

Update import in App.js to be correct

jasminepvo committed:
Build skeleton layout of home page

jasminepvo committed:
Complete navbar and main text sections styling

jasminepvo committed:
Complete main photos section and make responsive

jasminepvo committed:
Update html title, description and favicon

jasminepvo committed:
Remove empty css style

HotSoupRepos commented:
@jasminepvo This looks great! 
Three things: 
1) Can we please make the "Coming Soon" text not blurry?
2) Can we also add the Discord link to the Discord Button?
3) I know I said before we can keep the donate button, but can we please remove it for now, just until we have the donation profile fully set up?

HotSoupRepos commented on src/App.js:
```diff
@@ -1,20 +1,62 @@
-import { FaLinkedin } from 'react-icons/fa';
-import './global.css';
+// import { FaLinkedin } from "react-icons/fa";
```
Can we please delete this line?

HotSoupRepos commented on src/App.js:
```diff
@@ -1,20 +1,62 @@
+			</nav>
+			<div className="main">
+				<div className="main-text">
+					<h1>COMING SOON</h1>
```
Is there anyway to sharpen the text on this to make it not blurry?

HotSoupRepos commented on src/App.js:
```diff
@@ -1,20 +1,62 @@
+						details, all of our INCREDIBLE volunteers and links to download the
+						app!
+					</p>
+					<button className="btn-volunteer">
```
Can we please add an <a/> with the source linking to the Discord channel? Please let me know if you need help with that!

jasminepvo replied:
I set the link to never expires, is that okay?

HotSoupRepos commented on src/App.js:
```diff
@@ -1,20 +1,62 @@
+					<button className="btn-volunteer">
+						VOLUNTEER - JOIN THE DISCORD
+					</button>
+					<button className="btn-kickstarter">BACK OUR PROJECT</button>
```
Can we please actually hide this button, at least until we get a proper donation account going? This will happen soon, but I'd rather not have an inactive button in the live code. Sorry for changing that on you

jasminepvo replied:
No problem! Will comment it out

HotSoupRepos commented on src/global.css:
```diff
@@ -1,32 +1,188 @@
+	line-height: 7rem;
+	color: var(--primaryYellow);
+	text-shadow: 0px 4px 20px #000000;
+	filter: blur(2px);
```
if this is causing the blur, can we please remove it? It is making it just a tad bit hard to read

jasminepvo replied:
Done

jasminepvo replied:
Removed, is this better?
![Screen Shot 2022-10-30 at 11 26 26 AM](https://user-images.githubusercontent.com/99847030/198886951-ab9a8539-23f6-4837-88a3-3738fea39e7c.jpg)

HotSoupRepos commented on src/global.css:
```diff
@@ -1,32 +1,188 @@
+	justify-content: center;
+	align-items: center;
+}
+.main-text {
```
what do you think about adding in something like  https://travishorn.com/responsive-grid-in-2-minutes-with-css-grid-layout-4842a41420fe this for the resposiveness? I think it would be best to have a responsive layout, rather than adding in a lot of different media queries. You can make the whole layout a grid, and then when viewport screen resizes, the grid moves around to fitthe viewport. 

OR you can use bootstrap!  https://getbootstrap.com/docs/5.0/getting-started/introduction/

jasminepvo replied:
I will look into this, thanks for adding the resources!

jasminepvo committed:
Removed commented out code, remove blue from h1, hide kickstarter button, add discord link to volunteer button

jasminepvo committed:
Add bootstrap to project

jasminepvo committed:
Make responsive with mobile first approach

jasminepvo commented:
Applied all changes requested. Added boostrap and fixed responsiveness however I removed the arrow. Let me know your thoughts.

Mobile:
![HotSoupTemp-Mobile](https://user-images.githubusercontent.com/99847030/199359840-c650b5b6-7827-49f7-a5a3-c805e74589e7.png)

Desktop:
![HotSoupTemp-Desktop](https://user-images.githubusercontent.com/99847030/199360124-dd8f957e-08f8-4bad-818d-bed49df1070d.png)

jasminepvo committed:
Change from flex to grid, remove all media queries for layout, add 1 media query for font size of h1

jasminepvo committed:
Add margin to bottom of h3 'Here is a sneak peak of the app'

jasminepvo commented:
- change from flex to grid layout
- remove all media queries for layout
- add 1 media query for font size of h1

Please review and let me know.

DrummerDee commented on src/App.js:
```diff
@@ -1,20 +1,65 @@
-}
+const App = () => {
+	return (
+		<div className="body">
```
Please make this semantically correct,

jasminepvo replied:
Done

DrummerDee commented on src/App.js:
```diff
@@ -1,20 +1,65 @@
+						engineering is 100% volunteer based, helping seasoned engineering
+						sharpen their skills, new devs get some real-world experience (to
+						help land that first job that always seems to require experience),
+						all while using their skills to give back to the communities. <br />{" "}
```
Please remove breaks and use other forms of code to give it the styling that you prefer.

jasminepvo replied:
Remove breaks and made paragraphs

DrummerDee commented on src/global.css:
```diff
@@ -1,32 +1,114 @@
+	font-family: "Inder", "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+		"Lucida Sans", Arial, sans-serif;
+	padding: 1rem;
+	font-size: 16px;
```
Please change pixels to rem. Rem makes it easier when it comes to responsive design. I'm going to add a link to help you convert your pixels to REM so you don't have to guess. 

[Pixel to REM converter](https://nekocalc.com/px-to-rem-converter)

jasminepvo replied:
Done! Thanks for the link.

DrummerDee commented on src/global.css:
```diff
@@ -1,32 +1,114 @@
+/**************
+* MAIN PHOTOS *
+**************/
+.main-photos h3 {
```
I feel specificity make causes an issue here, could you use a karat to ensure that the targeted element is going to be affected?

Example .main-photos > h3. (This will ensure that the h3 in the .main-photos class will be affected and not any other h3.)

jasminepvo replied:
Added the karats

DrummerDee reviewed (changes requested):
This looks great Jasmine, would like to see some minor changes happening. I think you did amazing job learning the concept of Grid and implementing it into the application.

DrummerDee commented:
This looks great, some minor changes that I would suggest are adding some padding between the two phones, decreasing the font size on the "Coming Soon " as well as the HotSoup title in the header, changing the color or shadow on the "Here is a sneak.."(it will be hard to read for some people), center the body of text more. On mobile, it doesn't look that bad but while it's desktop, the text is a smidge too close to the margins. Thank you so much. If you have any questions at all, I  am here : )

jasminepvo committed:
Make semantically correct

jasminepvo committed:
Remove breaks and made paragraphs

jasminepvo committed:
Change pixels to rems

jasminepvo committed:
Added karats for specificity

jasminepvo committed:
Remove classname navbar in app and styles

jasminepvo committed:
Decrease size of navbar

jasminepvo committed:
Make changes to h3 for readability, adjust padding/margin

jasminepvo commented:
Thanks for the feedback! Made the adjustments suggested, let me know what else I can do 😄 

![HotSoupTempPage](https://user-images.githubusercontent.com/99847030/203458814-8e345fdd-625d-4b46-abc9-987a80645e25.png)
